model/src/main/java/com/basistech/rosette/apimodel/DocumentRequest.java

```diff
@@ -71,7 +68,7 @@
      */
     private final String contentType;
 
-    /**
+    /** @deprecated
      * @return request genre
      */
     private final String genre;
```

Comment:
Even though the field is private, should it also be marked as `@Deprecated` ?

Reply:
Yeah, cuz it will help us to keep track of it internally as well.